_meta/beat.yml

```diff
@@ -388,6 +388,12 @@ apm-server:
 
   # When enabling Jaeger integration, APM Server acts as Jaeger collector. It supports jaeger.thrift over HTTP
   # and gRPC. This is an experimental feature, use with care.
+  #
+  # WARNING: This configuration is deprecated, and will be removed in a future release.
```

Comment:
@axw the plan is to remove these config options in `8.0` right?

Reply:
Yes. I've updated the comment to state that.